### ISSUE TITLE
Unexpected Error with `pi()^x`: operator does not exist

### ISSUE THREAD

**Issue by DominicWuest** [apache/age#877]
When running the following query:
```cypher
RETURN pi()^0
```

I encountered the following error:
```
operator does not exist: double precision ^ agtype
```

I believe the query mentioned above is semantically and syntactically correct and thus no error should be thrown here.

I encountered this issue when testing queries against the **apache/age:PG13_latest** docker image.

### Steps to reproduce
Run the following query and observe it throws an error:
```cypher
RETURN pi()^0
```

### Expected behavior
The query should run successfully

### Actual behavior
The query fails with the error message `operator does not exist: double precision ^ agtype`.

**Comment by mohayu22** [comment]
This problem has been resolved in PR #823
AGE 1.3.0 has not been updated yet with this commit

```
SELECT * FROM cypher('test', $$
  RETURN pi()^2
$$) AS (res agtype);
       res        
------------------
 9.86960440108936
(1 row)

```
```
SELECT * FROM cypher('test', $$
  RETURN rand()^2
$$) AS (res agtype);
        res        
-------------------
 0.028840664980237
(1 row)

```

**Comment by DominicWuest** [comment]
I see, thanks for the info @mohayu22!
